### PR TITLE
feature: add date route

### PR DESCRIPTION
**Pull request by savioferraz**

### CODE REVIEW

**Comment by denise-techlead** [comment]
Great! Please wait a moment while I review your code. 👀

**Comment by denise-techlead** [comment]
The unit tests did not pass. This means there is something wrong with your solution. Fix it and create a new commit on this branch to update this Pull Request. 👀

File: src/services/dateService.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 const getCurrentDate = () => {
-    // Add your solution here!
+  const date = new Date();
+  const formatedDate = `${date.getDate().toString()}/${date
```
Comment (leticiasassaki) [inline src/services/dateService.js]: Olá, @savioferraz! Tudo bem?
Tivemos uma mudança nessa CS e corrigimos o texto da Denise.
O retorno esperado seria: `yyyy-mm-dd`

**Comment by denise-techlead** [comment]
The unit tests did not pass. This means there is something wrong with your solution. Fix it and create a new commit on this branch to update this Pull Request. 👀

**Review by denise-techlead — APPROVED** [review]
Everything seems to be fine over here. Congratulations!

**Pull request merged**